src/modules/notifications/actions/set-notification-text.js

```diff
@@ -421,27 +428,23 @@ export default function setNotificationText(notification, callback) {
       case "SENDETHER":
         notification.title = "Send ETH";
         if (!notification.description && notification.params) {
-          notification.description =
-            "Send " +
-            formatEther(notification.params.etherToSend).formatted +
-            " ETH to " +
-            notification.params.to;
+          notification.description = `Send ${
+            formatEther(notification.params.etherToSend).formatted
+          } ETH to ${notification.params.to}`;
         }
         break;
       case "SENDREPUTATION":
         notification.title = "Send REP";
         if (!notification.description && notification.params) {
-          notification.description =
-            "Send " +
-            formatRep(notification.params.reputationToSend).formatted +
-            " REP to " +
-            notification.params._to;
+          notification.description = `Send ${
+            formatRep(notification.params.reputationToSend).formatted
+          } REP to ${notification.params._to}`;
         }
         break;
 
       default: {
         const result = notification.params.type.replace(/([A-Z])/g, " $1");
-        notification.title = result.charAt(0).toUpperCase() + result.slice(1);
+        notification.title = result;
```

Comment:
Why did this get removed? Isn't this needed to make the first letter of the title uppercase since it's removing the camel casing?